### PR TITLE
Gregor metadata airtable update

### PR DESCRIPTION
**Pull request by hanars**

### CODE REVIEW

File: seqr/views/utils/export_utils.py [hunk 1]
```diff
@@ -75,10 +75,9 @@ def _format_files_content(files,  file_format='csv', add_header_prefix=False, bl
             header_display = ['{}-{}'.format(str(header_tuple[0]).zfill(2), header_tuple[1]) for header_tuple in
                               enumerate(header)]
             header_display[0] = header[0]
-        content = DELIMITERS[file_format].join(header_display) + '\n'
         content_rows = [[row.get(key) or blank_value for key in header] for row in rows]
-        content += '\n'.join([
-            DELIMITERS[file_format].join(row) for row in content_rows
+        content = '\n'.join([
```
Comment (hanars) [inline seqr/views/utils/export_utils.py:79]: fixes a bug when the table is empty

Comment (ShifaSZ) [reply seqr/views/utils/export_utils.py:79]: Add a test for an empty table?

**Review by ShifaSZ — CHANGES_REQUESTED** [review]

File: seqr/views/utils/export_utils.py (hunk 1 above)
Comment (hanars) [reply seqr/views/utils/export_utils.py:79]: out of scope, I just happened to notice it here but this endpoint can't return empty tables

File: seqr/views/apis/report_api.py [hunk 2]
```diff
@@ -684,30 +688,59 @@ def _get_sample_airtable_metadata(sample_ids, user, include_collaborator=False):
...
+RNA_ONLY = EXPERIMENT_RNA_TABLE_AIRTABLE_FIELDS + READ_RNA_TABLE_AIRTABLE_FIELDS + ['reference_assembly_uri']
+DATA_TYPE_OMIT = {
+    'wgs': ['targeted_regions_method'] + RNA_ONLY, 'wes': RNA_ONLY, 'rna': [
+        'targeted_regions_method', 'target_insert_size', 'mean_coverage', 'aligned_dna_short_read_file',
+        'aligned_dna_short_read_index_file',
+    ],
+}
+NO_DATA_TYPE_FIELDS = {
+    'targeted_region_bed_file', 'reference_assembly', 'analysis_details', 'percent_rRNA', 'percent_mRNA',
+    'alignment_software_dna',
```
[the remark this answers is not in the document]
Comment (hanars) [reply seqr/views/apis/report_api.py:730]: Why? We don't for the other 7 tables already in this report, why would we do it for this new one

File: seqr/views/apis/report_api.py [hunk 3]
```diff
@@ -791,18 +824,34 @@ def gregor_export(request):
...
         projectcategory__name='GREGoR',
     )
-    individuals = Individual.objects.filter(
-        sample__in=get_search_samples(projects, active_only=False),
-    ).distinct().prefetch_related('family__project', 'mother', 'father')
+    sample_types = Sample.objects.filter(individual__family__project__in=projects).values_list('individual_id', 'sample_type')
+    individual_data_types = defaultdict(set)
+    for individual_db_id, sample_type in sample_types:
+        individual_data_types[individual_db_id].add(sample_type)
+    individuals = Individual.objects.filter(id__in=individual_data_types).prefetch_related(
```
[the remark this answers is not in the document]
Comment (hanars) [reply seqr/views/apis/report_api.py:831]: thats just how iterating a dictionary in python3 works

**Review by ShifaSZ — APPROVED** [review]
